test/dialect/postgresql/test_compiler.py

```diff
@@ -936,7 +984,19 @@ def test_create_index_with_with(self):
             schema.CreateIndex(idx3),
             "CREATE INDEX test_idx3 ON testtbl "
             "USING gist (data) "
-            "WITH (buffering = off)",
+            "WITH (buffering = 'off')",
```

Comment:
passing this as string seems accepted, but this kinda seems like a breaking change, so not sure this should be a v2 change.

for 2.1 should be ok, but maybe not on 2.0.

what do you think mike @zzzeek

Reply:
works equally well on versions 11-15

https://sqlize.online/sql/psql11/e3e71f14c6fe2cc19090c22ebad1c2fb/
https://sqlize.online/sql/psql12/e3e71f14c6fe2cc19090c22ebad1c2fb/
https://sqlize.online/sql/psql13/e3e71f14c6fe2cc19090c22ebad1c2fb/
https://sqlize.online/sql/psql14/e3e71f14c6fe2cc19090c22ebad1c2fb/
https://sqlize.online/sql/psql15/e3e71f14c6fe2cc19090c22ebad1c2fb/

Reply:
This case sure, but this does not make it not a breaking change:
Something like `{'foo': 'true', 'bar': 'null'}` would likely not work now, since true and null would be passed as strings, not true the bool and null the missing value

Reply:
Yes, from that perspective it's a breaking change
Then we'll wait for 2.1

Reply:
-1 on quoting "off" , the syntax at https://www.postgresql.org/docs/current/sql-createindex.html shows it without quotes.

Reply:
string is also accepted. I too would be for taking string to mean sql text as is (current behavior)
If people want to pass arbitrary things we can document that [`literal()`](https://docs.sqlalchemy.org/en/20/core/sqlelement.html#sqlalchemy.sql.expression.literal) will be properly compiled